syntax="proto3";

package transport_proto;

message RouteSettings {
	int32 bus_wait_time = 1;
	double bus_velocity = 2;
}